bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_gnn.py

```diff
@@ -0,0 +1,259 @@
+# Authors:
+# * Sanjiban Sengupta 01/2023
+# * Lorenzo Moneta    01/2023
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from .. import pythonization
+from cppyy import gbl as gbl_namespace
+
+def getActivationFunction(model):
+    function = model._activation.__name__
+    if function == 'relu':
+        return gbl_namespace.TMVA.Experimental.SOFIE.Activation.RELU
+    else:
+        return gbl_namespace.TMVA.Experimental.SOFIE.Activation.Invalid
+
+def make_mlp_model(gin, model, target, type):
+    num_layers = len(model._layers)
+    activation = getActivationFunction(model)
+    upd = gbl_namespace.TMVA.Experimental.SOFIE.RFunction_MLP(target, num_layers, activation, model._activate_final, type)
+    kernel_tensor_names = gbl_namespace.std.vector['std::string']()
+    bias_tensor_names   = gbl_namespace.std.vector['std::string']()
+
+    for i in range(0, 2*num_layers, 2):
+        bias_tensor_names.push_back(model.variables[i].name)
+        kernel_tensor_names.push_back(model.variables[i+1].name)
+    val = gbl_namespace.std.vector['std::vector<std::string>']()
+    val.push_back(kernel_tensor_names)
+    val.push_back(bias_tensor_names)
+    upd.AddInitializedTensors(val)
+    gin.createUpdateFunction(upd)
+
+def add_layer_norm(module_layer, model_block):
```

Comment:
Include docstrings of this function and document the input arguments. Same as other comments above

Reply:
made the change, thanks!